code/modules/cm_marines/Donor_Items.dm

```diff
@@ -40,389 +41,6 @@
 	//AND THIS LINE
 //END SUIT TEMPLATE
 
-/obj/item/clothing/suit/storage/marine/fluff/santa
-	name = "Santa's suit"
-	desc = "Festive!  DONOR ITEM"
-	icon_state = "santa"
-	item_state = "santa"
-
-/obj/item/clothing/suit/storage/marine/fluff/armorammo
-	name = "marine armor w/ ammo"
-	desc = "A marine combat vest with ammunition on it.  DONOR ITEM"
-	icon_state = "bulletproofammo"
-	item_state = "bulletproofammo"
-
-/obj/item/clothing/suit/storage/marine/fluff/cia
-	name = "CIA jacket"
-	desc = "An armored jacket with CIA on the back.  DONOR ITEM"
-	icon_state = "cia"
-	item_state = "cia"
-
-/obj/item/clothing/suit/storage/marine/fluff/obey
-	name = "Black Ops Ablative Armor Vest"
-	desc = "Some fancy looking armor.  DONOR ITEM"
-	icon_state = "armor_reflec"
-	item_state = "armor_reflec"
-
-/obj/item/clothing/suit/storage/marine/fluff/sas5
-	name = "Juggernaut Armor"
-	desc = "Some fancy looking armor. DONOR ITEM"
-	icon_state = "rig-syndi"
-	item_state = "syndie_hardsuit"
-
-/obj/item/clothing/suit/storage/marine/fluff/penguin
-	name = "Trenchcoat"
-	desc = "An 18th-century trenchcoat. Someone who wears this means serious business.  DONOR ITEM"
-	icon_state = "detective"
-	item_state = "det_suit"
-	blood_overlay_type = "coat"
-
-/obj/item/clothing/suit/storage/marine/fluff/wright
-	name = "Swat Armor"
-	desc = "Some fancy looking armor. DONOR ITEM"
-	icon_state = "deathsquad"
-	item_state = "swat_suit"
-
-/obj/item/clothing/suit/storage/marine/fluff/tyran
-	name = "Swat Armor"
-	desc = "Some fancy looking armor. DONOR ITEM"
-	icon_state = "deathsquad"
-	item_state = "swat_suit"
-
-/obj/item/clothing/suit/storage/marine/fluff/tristan
-	name = "Sciency Teleport Armor"
-	desc = "Some fancy looking armor, with lots of lights and buttons.  DONOR ITEM"
-	icon_state = "reactive"
-	item_state = "reactive"
-	blood_overlay_type = "armor"
-
-/obj/item/clothing/suit/storage/marine/fluff/SAS
-	name = "Legion Armor"
-	desc = "This armor was custom made to resemble the small growing Legion within the galaxy started by one man slowly making its way to becoming a larger Corperation.  DONOR ITEM."
-	item_state = "ncrjacket"
-	icon_state = "ncrjacket"
```

Comment:
technicaly you can have this if this is the f13 one, f13 one is open sauce

Reply:
If it stays in it should just be an unused icon, not a donor item.

Reply:
if any of these deserve to be saved, they can be added as loadout items in a separate PR.